hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/audit/TestOzoneAuditLogger.java

```diff
@@ -143,7 +150,35 @@ public void notLogReadEvents() throws IOException {
     verifyNoLog();
   }
 
-  private void verifyLog(String expected) throws IOException {
+  /**
+   * Test to verify if multiline entries can be checked.
+   */
+
+  @Test
+  public void messageIncludesMultilineException() throws IOException {
+    String exceptionMessage = "Dummy exception message";
+    TestException testException = new TestException(exceptionMessage);
+    AuditMessage exceptionAuditMessage =
+        new AuditMessage.Builder()
+            .setUser(USER)
+            .atIp(IP_ADDRESS)
+            .forOperation(DummyAction.CREATE_VOLUME)
+            .withParams(PARAMS)
+            .withResult(FAILURE)
+            .withException(testException).build();
+    AUDIT.logWriteFailure(exceptionAuditMessage);
+    verifyLog(
+        "ERROR | OMAudit | user=john | "
+            + "ip=192.168.0.1 | op=CREATE_VOLUME "
+            + "{key1=value1, key2=value2} | ret=FAILURE",
+        "org.apache.hadoop.ozone.audit."
+            + "TestOzoneAuditLogger$TestException: Dummy exception message",
+        "at org.apache.hadoop.ozone.audit.TestOzoneAuditLogger"
+            + ".messageIncludesMultilineException"
+            + "(TestOzoneAuditLogger.java:160) [test-classes/:?]");
```

Comment:
`at ...TestOzoneAuditLogger.java:160`

This is very fragile.  Any code change above line 160 (as benign as changing imports or whitespace) might break it.  Can you please omit the check for line number?

Reply:
@adoroszlai indeed, thank you for pointing that out. Fixed in latest commit.